hashbang/welcome.pre

```diff
@@ -0,0 +1,9 @@
+   _  _    __
+ _| || |_ |  |  Welcome to #!. This network has three rules:
+|_  __  _||  |
+ _| || |_ |  |  1. When people need help, teach. Don't do it for them
+|_  __  _||__|  2. Don't use our resources for closed source projects
+  |_||_|  (__)  3. Be excellent to each other
+
+ Things to explore:
```

Comment:
I feel like "Things to explore:" might be better duplicated in welcome.{not,t}mux